pkg/prove/proof.go

```diff
@@ -193,3 +246,96 @@ func splitIntoRows(squareSize uint64, s []shares.Share) [][]shares.Share {
 	}
 	return rows
 }
+
+// SharesInclusion generates an nmt inclusion proof for a set of shares to the data root.
+// expects the shares range to be pre-validated.
```

Comment:
[question] can you please clarify what you mean by pre-validated?

```suggestion
// Expects the share range to be pre-validated.
```

Reply:
4595dd2